packages/saber-plugin-google-analytics/index.js

```diff
@@ -4,11 +4,12 @@ const ID = 'google-analytics'
 
 exports.name = ID
 
-exports.apply = (api, { trackId = false } = {}) => {
+exports.apply = (api, { trackId = false, respectDoNotTrack = true } = {}) => {
   api.hooks.chainWebpack.tap(ID, config => {
     config.plugin('constants').tap(([options]) => [
       Object.assign(options, {
-        __GA_TRACK_ID__: JSON.stringify(trackId)
+        __GA_TRACK_ID__: JSON.stringify(trackId),
+        respectDoNotTrack
```

Comment:
Probably there's no reason to not to respect `doNotTrack` 😄

Reply:
In case people need more precise traffic data ( though it's rare 😶)
I have privacy concerns too, so I put this default to true. Make it always true is just a step further and that's fine!